app/views/layouts/application.html.erb

```diff
@@ -144,7 +150,7 @@
 
   <%# Include logo-dark (used for printing) here because for just-in-time ajax loading %>
   <%# (e.g. forms) it may not load in time. %>
```

Comment:
Took me awhile to figure out why this was here but hidden. Can you add to the comment to indicate the purpose is to force the pre-loading of the image so that when it's included elsewhere, it's already available to the browser?